xml/System.Net/Cookie.xml

```diff
@@ -113,11 +113,11 @@
   
  The `value` parameter for a <xref:System.Net.Cookie> must not be a `null` reference (Nothing in Visual Basic). The following characters are reserved and cannot be passed in the `value` parameter unless the string passed in the `value` parameter is enclosed in double quotes: semicolon, comma. So the following example constructor would succeed but when you try to add this <xref:System.Net.Cookie> to a <xref:System.Net.CookieContainer> instance with the <xref:System.Net.CookieContainer.Add%2A> or <xref:System.Net.CookieContainer.Add%2A> methods the operation will fail and throw an exception:  
   
- `System.Net.Cookie cookie = new System.Net.Cookie("contoso", "123,456", "");`  
+ `System.Net.Cookie cookie = new System.Net.Cookie("contoso", "123,456", "", "contoso.com");`  
```

Comment:
Given that this is the documentation for the #ctor(String,String) constructor, it would be best for the code to call that constructor, rather than a different constructor.